ISSUE_TEMPLATE.md

```diff
@@ -1,4 +1,10 @@
 <!--
+  If this is a performance issue report, please use the VS Code Issue reporter via command palette (f1 -> Help: Report Issue).
+  Set the field 'This is a' to be 'Performance Issue', the field 'File on' to be 'An Extension', and set 'Extension' to be 'C/C++'.
```

Comment:
How does this differ from non-performance issues?

Reply:
The VS Code issue reporter will add information such as system info, currently running processes, workspace status, and extensions installed.